docs/dev/build_android.md

```diff
@@ -5,54 +5,121 @@ This article describes how to build OpenVINO for Android operating systems.
 
 ## How to build
 
-1. Download and unpack [Android NDK](https://developer.android.com/ndk/downloads). Let's assume that `~/Downloads` is used as a working folder.
+### Create work directory and set environment variables for building
   ```sh
-  cd ~/Downloads
-  wget https://dl.google.com/android/repository/android-ndk-r20-linux-x86_64.zip
-
-  unzip android-ndk-r20-linux-x86_64.zip
-  mv android-ndk-r20 android-ndk
+  mkdir openvino-android
+  export OPV_HOME_DIR=${PWD}/openvino-android
+  export CURRENT_ANDROID_ABI=arm64-v8a
+  export CURRENT_ANDROID_PLATFORM=30
+  export CURRENT_ANDROID_STL=c++_shared
   ```
+  * `ANDROID_ABI` specifies target architecture:
+     * `x86_64` for x64 build
+     * `armeabi-v7a with NEON` for ARM with NEON support
+     * `arm64-v8a` for ARM 64 bits
+  * `ANDROID_PLATFORM` - Android API version
+  * `ANDROID_STL` specifies that shared C++ runtime is used. 
 
-2. Create a build folder:
-  ```sh
-  mkdir build
-  ```
+### Download and unpack Android packages  and 
+* Download and unpack [Android NDK](https://developer.android.com/ndk/downloads)
+```sh
+  wget https://dl.google.com/android/repository/android-ndk-r26d-linux.zip --directory-prefix $OPV_HOME_DIR
 
-3. Change working directory to `build` and run `cmake` to create makefiles. Then run `make`.
-  ```sh
-  cd build
+  unzip $OPV_HOME_DIR/android-ndk-r26d-linux.zip -d $OPV_HOME_DIR
+  mv $OPV_HOME_DIR/android-ndk-r26d $OPV_HOME_DIR/android-ndk
+  export ANDROID_NDK_PATH=$OPV_HOME_DIR/android-ndk
+```
+* Download and unpack [Android SDK Platform Tools](https://developer.android.com/tools/releases/platform-tools)
+```sh
+  wget https://dl.google.com/android/repository/platform-tools-latest-linux.zip --directory-prefix $OPV_HOME_DIR
 
-  cmake .. \
-    -DCMAKE_TOOLCHAIN_FILE=~/Downloads/android-ndk/build/cmake/android.toolchain.cmake \
-    -DANDROID_ABI=x86_64 \
-    -DANDROID_PLATFORM=21 \
-    -DANDROID_STL=c++_shared
+  unzip $OPV_HOME_DIR/platform-tools-latest-linux.zip -d $OPV_HOME_DIR
+  export ANDROID_TOOLS_PATH=$OPV_HOME_DIR/platform-tools
+  ```
 
-  make --jobs=$(nproc --all)
+_For Windows and Mac operating systems, download and unpack steps are similar_
+
+### Build and install OneTBB™
+To improve parallelism performance of the OpenVINO™ library using OneTBB, it is required to separately build OneTBB for a specific version of andoird-ndk:
+  ```bash
+  # Clone OneTBB™ repository 
+  git clone --recursive https://github.com/oneapi-src/oneTBB $OPV_HOME_DIR/one-tbb
+  # Create build and install directory 
+  mkdir $OPV_HOME_DIR/one-tbb-build $OPV_HOME_DIR/one-tbb-install
+  # Configure OneTBB™ CMake project 
+  cmake -S $OPV_HOME_DIR/one-tbb \
+        -B $OPV_HOME_DIR/one-tbb-build \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INSTALL_PREFIX=$OPV_HOME_DIR/one-tbb-install \
+        -DCMAKE_TOOLCHAIN_FILE=$ANDROID_NDK_PATH/build/cmake/android.toolchain.cmake \
```

Comment:
can we set `CMAKE_TOOLCHAIN_FILE` globally? so, it can be used by both TBB and OpenVINO

Reply:
@ilya-lavrenov corrected